### ISSUE TITLE
Tail of the condor logs in wmstats with plenty of newlines

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#7798]
Just noticed it in my VM nodes (alan-cloud1). The condor.err file contains
```
ERROR  : Container not found: /cvmfs/singularity.opensciencegrid.org/bbockelm/cms:rhel6
ABORT  : Retval = 255
```

while wmstats reports something like
```
Could not find jobReport
Adding end of condor.4612.128.out to error message:
Adding end of condor.4612.128.err to error message:
E
R
R
O
R
 
 
:
 
C
o
```

probably there is a place expecting a list of strings but instead it's getting a plain string.

**Comment by amaltaro** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Tail of the condor logs in wmstats with plenty of newlines](https://trello.com/c/5FBqLaV3/28-tail-of-the-condor-logs-in-wmstats-with-plenty-of-newlines)

**Comment by vlimant** [comment]
+1

**Comment by ticoann** [comment]
fix is included in #7933
